src/common/mobileNav.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './mobileNav.css';
+
+function MobileNav(props) {
```

Comment:
These navbars look very nice - could we make a Phab issue to extract these to a separate library for long term use?

Reply:
I'll make a ticket on Phab!

Reply:
https://phab.sunrisemovement.org/T29